fern/pages/python_api/python-api-rmm-pylibrmm.md

```
@@ -0,0 +1,703 @@
+
+_Source: `python/rmm/rmm/pylibrmm/device_buffer.pyi:56`_
+
+### `copy_to_host`
+
+```python
+def copy_to_host(self, ary: None = None, stream: Stream = ...) -> NDArray[np.uint8]:
+```
+
+_Source: `python/rmm/rmm/pylibrmm/device_buffer.pyi:58`_
+
+### `copy_to_host`
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)